GitTfs/Commands/Clone.cs

```diff
@@ -113,21 +111,18 @@ public int Run(string tfsUrl, string tfsRepositoryPath, string gitRepositoryPath
             bool errorOccurs = false;
             try
             {
-                if (withBranches && initBranch != null)
-                    fetch.IgnoreBranches = false;
-
                 if (tfsRepositoryPath == GitTfsConstants.TfsRoot)
-                    fetch.IgnoreBranches = true;
+                    fetch.BrancheStrategy = BranchStrategy.None;
```

Comment:
S'il te plaît, spell `BrancheStrategy` et `BranchStrategy` the same façon? :wink: